helm/librechat/templates/configmap-env.yaml

```diff
@@ -0,0 +1,17 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ include "librechat.fullname" $ }}-configenv
+data:
+  {{- if (index .Values "librechat-rag-api" "enabled") }}
+  RAG_API_URL: http://{{ include "rag.fullname" (index .Subcharts "librechat-rag-api") | lower }}.{{ .Release.Namespace | lower }}.svc.cluster.local:8000 
+  {{- end }}
+  {{- if .Values.librechat.configEnv }}
```

Comment:
It is just cosmetic, but I would put them more likely at the end than in between